11/11-1/exercise3.test.js

```diff
@@ -0,0 +1,51 @@
+const users = {
+  4: { name: 'Mark' },
+  5: { name: 'Paul' },
+};
+
+const findUserById = id => {
+  return new Promise((resolve, reject) => {
+    if (users[id]) {    
+      return resolve(users[id]);
+    }
+
+    return reject({ error: 'User with ' + id + ' not found.' });
+  });
+};
+
+const getUserName = userID => {
+  const test = asyncfindUserById(userID).then(user => user.name);
+  return findUserById(userID).then(user => user.name);
+};
+
+console.log('findUserById(4)',findUserById(4));
+console.log(findUserById(4).then(animal => animal));
+console.log(findUserById(1).catch(error => error));
```

Comment:
Entendo que em um ambiente de estudo, console.log é uma ferramenta muito poderosa, mas cuidado quando for subir seu projeto.

Reply:
Voce esta certo. Preciso prestar mais atenção nisso